nvmeadm/src/nvmf_subsystem.rs

```diff
@@ -23,26 +23,114 @@ pub struct Subsystem {
     pub state: String,
     /// The transport type being used (tcp or RDMA).
     pub transport: String,
-    /// Address contains traddr=X,trsvcid=Y.
-    pub address: String,
+    /// Address contains a comma-separated list of `SubsystemAddrToken`.
+    /// Example: traddr=X,trsvcid=Y.
+    pub address: SubsystemAddr,
     /// Serial number.
     pub serial: String,
     /// Model number.
     pub model: String,
 }
 
-/// Wrapper structure that creates subsystem addr string.
+const TR_ADDR: &str = "traddr";
+const TR_SVC_ID: &str = "trsvcid";
+const SRC_ADDR: &str = "src_addr";
+
+#[derive(Debug, Clone)]
+#[allow(unused)]
```

Comment:
Is this needed?

Reply:
Yes, because we're not using unknown atm